alerting/build.gradle

```diff
@@ -120,4 +120,4 @@ run {
     distribution = "oss-zip"
 }
 
-apply from: '../build-tools/rpmbuild.gradle'
+apply from: '../build-tools/pkgbuild.gradle'
```

Comment:
I don't see this file renamed in the diff.

Reply:
It says "build-tools/rpmbuild.gradle → build-tools/pkgbuild.gradle" for me in the title of the second diff on this page. Is this what you're looking for?